cspice/src/coordinates.rs

```diff
@@ -1,16 +1,32 @@
 //! Functions for converting between different types of coordinates.
 use crate::spice_unsafe;
-use crate::vector::Vector3D;
 use cspice_sys::{azlrec_c, recazl_c, reclat_c, recrad_c, SpiceBoolean, SpiceDouble};
-use derive_more::{Deref, DerefMut, From, Into};
+use derive_more::{Deref, DerefMut, Into};
 
 /// Rectangular coordinates
-#[derive(Copy, Clone, Debug, Default, PartialEq, From, Into, Deref, DerefMut)]
-pub struct Rectangular(Vector3D);
+#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, PartialEq, Into, Deref, DerefMut)]
+pub struct Rectangular {
+    #[deref]
+    #[deref_mut]
```

Comment:
I don't really see the need for the deref here - seems a bit confusing? I normally only use deref when there is a single field.

Also what does the derive `Into` do here?

Reply:
Yes I agree it's confusing and not necessary with your proposed solution. Thanks!